src/proj.def

```diff
@@ -151,3 +151,6 @@ EXPORTS
     pj_calc_ellipsoid_params       @135
     pj_chomp                       @136
     pj_shrink                      @137
+
+    pj_approx_2D_trans             @138
```

Comment:
I must admit, I don't fully understand how this file works. Is it really necessary to include these functions here? They don't go into the public API, so do they really need entry points in the DLL?

Reply:
Yes, because even though not in the public API, they are part of the "private API", i.e. functionality used in the PROJ project provided apps: The pj_chomp, pj_shrink and pj_approx... funcs are used by gie.c, and hence need to be exposed in the library. `pj_calc_ellipsoid_params` apparently is not used in any apps anymore, so it can go. Probably a thing for another PR